res/xml/qwertz.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?> 
+<keyboard>
+  <row>
+    <key key0="q" key1="esc" key2="1" key3="\@" key4="!"/>
+    <key key0="w" key1="^" key2="2" key3="&quot;" key4="°"/>
+    <key key0="e" key1="§" key2="3" key4="€"/>
+    <key key0="r" key2="4" key3="$"/>
+    <key key0="t" key2="5" key3="%"/>
+    <key key0="z" key2="6" key3="&amp;" key4="{"/>
+    <key key0="u" key2="7" key3="ü" key4="}"/>
+    <key key0="i" key1="(" key2="8" key4="["/>
+    <key key0="o" key1=")" key2="9" key3="ö" key4="]"/>
+    <key key0="p" key1="=" key2="0" key3="~" key2="\?"/>
+  </row>
+  <row>
+    <key shift="0.5" key0="a" key1="tab" key2="`" key3="ä"/>
```

Comment:
Are `ä` and `ü` useful if they can already be typed with the `trema` accent ?
`é` and `è` are very common in French but I didn't feel the need to add a dedicated key for them, accents are fast enough for me.

Reply:
If we have proper äöü and ÄÖÜ, we do not need trema - I've never needed it, except for writing Joyeux Noel in French... Please, we do not see the umlauts as accents, they're independent letters of the alphabet.

Reply:
I understand that they are independent letters, it's the same in french. But it's not how they are presented in the keyboard, on purpose.
This concept is called a dead key: https://en.wikipedia.org/wiki/Dead_key